modules/logging-forward-splunk.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+// cluster-logging-external.adoc
+//
+
+:_content-type: PROCEDURE
+[id="logging-forward-splunk_{context}"]
+= Forwarding logs to Splunk
+
+You can forward logs to the link:https://docs.splunk.com/Documentation/Splunk/9.0.0/Data/UsetheHTTPEventCollector[Splunk HTTP Event Collector (HEC)] in addition to, or instead of, the internal default {product-title} log store.
+
+[NOTE]
+====
+Using this feature with Fluentd is not supported.
+====
+
+.Prerequisites
+* Red Hat OpenShift Logging Operator 5.6 and higher
+* ClusterLogging instance with vector specified as collector
+* Base64 encoded Splunk HEC token
+
+.Procedure
+
+. Create a secret using your Base64 encoded Splunk HEC token.
+[source,terminal]
```

Comment:
```suggestion
[source,terminal]
+
```
The formatting of the list is broken because of the missing `+`:
![image](https://user-images.githubusercontent.com/86380613/219769327-771251a0-e1c9-49e6-94dc-502e5f45a0c8.png)